### ISSUE TITLE
Enforce OpenGL mode when plugin is activated

### ISSUE THREAD

**Issue by keesverruijt** [opencpn-radar-pi/radar_pi#95]
Since the plugin doesn't work without it, we should try enabling it when the user enables our plugin.

**Comment by douwefokkema** [comment]
Good idea. I also forget it now and then. But is there an API  or some other trick to get at it?
